Resources/Maps/centcomm.yml

```diff
@@ -25435,7 +25435,7 @@ entities:
     parent: 63
     type: Transform
 - uid: 2526
-  type: SignSecurearea
```

Comment:
Seeing as you're already renaming this, you might as well capitalize the "Area"

Reply:
Makes sense, changed thanks